packages/cli/templates/platform-app-tailwind/src/App.tsx

```diff
@@ -0,0 +1,124 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import React, { useEffect, useState } from 'react';
+import {
+    Button,
+    ButtonEmphasis,
+    Card,
+    LoadingCircle,
+    LoadingCircleSize,
+    LoadingCircleStyle,
+    Stack,
+    Text,
+} from '@frontify/fondue';
+import { PlatformAppContext, usePlatformAppBridge } from '@frontify/app-bridge';
+
+type AssetResource = {
+    type: string;
+    id: string;
+    title?: string;
+    previewUrl?: string;
+    downloadUrl?: string;
+    filename?: string | null;
+    sourceUrl?: string;
+    html?: string;
+};
+
+const App = () => {
+    const appBridge = usePlatformAppBridge();
+
+    const [imageSrc, setImageSrc] = useState<string | undefined>(undefined);
+    const [assetRessources, setAssetRessources] = useState<AssetResource>();
+    const [context, setContext] = useState<PlatformAppContext>([]);
+    const [reUploadCounter, setReUploadCounter] = useState<number>(0);
+    const [uploading, setUploading] = useState<boolean>(false);
+
+    useEffect(() => {
+        (async () => {
+            if (!appBridge) {
+                return;
+            }
+
+            const currentContext = appBridge.context().get();
+            setContext(currentContext);
+
+            if (currentContext?.type === 'ASSET_ACTION') {
+                const assetRessources = await appBridge.api({
+                    name: 'getAssetResourceInfo',
+                    payload: { assetId: currentContext.assetId },
+                });
+
+                setAssetRessources(assetRessources);
+                setImageSrc(assetRessources.previewUrl);
+            }
+        })();
+    }, [appBridge]);
+
+    const onButtonClick = async () => {
+        if (!appBridge || !assetRessources?.downloadUrl) {
+            return;
+        }
+
+        setUploading(true);
+        await appBridge.api({
+            name: 'createAsset',
+            payload: {
+                data: assetRessources.downloadUrl,
+                filename: 'test.jpeg',
+                parentId: appBridge?.context('parentId').get(),
+            },
+        });
+        setReUploadCounter(reUploadCounter + 1);
+        setUploading(false);
+    };
+
+    return (
+        <div className="tw-flex tw-justify-center tw-flex-col tw-items-center tw-h-[100vh]">
```

Comment:
```suggestion
        <div className="flex justify-center flex-col items-center h-[100vh]">
```
As it is not part of the main app?